docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md

```diff
@@ -7,7 +7,7 @@ approvers:
 {% capture overview %}
 This tutorial shows you how to deploy a WordPress site and a MySQL database using Minikube. Both applications use PersistentVolumes and PersistentVolumeClaims to store data. 
 
-A [PersistentVolume](/docs/concepts/storage/persistent-volumes/) (PV) is a piece of storage in the cluster that has been provisioned by an administrator, and a [PersistentVolumeClaim](/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) (PVC) is a set amount of storage in a PV. PersistentVolumes and PersistentVolumeClaims are independent from Pod lifecycles and preserve data through restarting, rescheduling, and even deleting Pods. 
+A [PersistentVolume](/docs/concepts/storage/persistent-volumes/) (PV) is a piece of storage in the cluster that has been provisioned by an administrator, and a [PersistentVolumeClaim](/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) (PVC) is a set amount of storage in a PV. PersistentVolumes and PersistentVolumeClaims are independent from Pod lifecycles and preserve data through restarting, rescheduling, and even deleting Pods.  PersistentVolumes can be automatically provisioned by Kubernetes using a [StorageClass](/docs/concepts/storage/storage-classes/).
```

Comment:
Not your change. But I feel the explanation of PVC and PV is not very clear. 
1. a PVC is "a set amount of storage in a PV"?
2. PV is a piece of storage in the cluster has been provisioned by an administrator, or automatically provisioned by Kubernetes using a [StorageClass]?

Reply:
+1 especially about point #1